### ISSUE TITLE
Choose tool for data anlaysis and visualization

### ISSUE THREAD

**Issue by DelnaMathew** [professor-greebie/SENG8080-3-field_project#115]
We have to decide which tool to be used for data analysis and visualization. 
Some options are Jupiter Notebook, Tableau, and Power BI.
Any suggestions?

**Comment by svitolopes** [comment]
I like the idea of using Tableau. As we are learning about it, using it would be interesting.

**Comment by maulikpatel30** [comment]
What would I suggest is we should use Jupyter Notebook for EDA and for Visualization we can use Tableau as suggested by @svitolopes

**Comment by Ashish8128** [comment]
Also we can use R studio for exploratory data analysis as we have learned that in our previous semester and for data visualization can use Tableau as suggested by @svitolopes and @maulikpatel30

**Comment by varunbansal313** [comment]
I think Jupyter Notebook is a good option, as using Tableau Public we can integrate with Jupyter Notebook. Also using Jupyter Notebook, we can refine our Python coding skills. What say?

**Comment by AniketTiwari19** [comment]
I agree with @varunbansal313 as using Jupyter Notebook with Tableau allows for seamless integration of code-based data exploration and analysis in Jupyter, and the creation of interactive visualizations and dashboards in Tableau.

**Comment by dhruvi-249** [comment]
I agree with @maulikpatel30 and @varunbansal313 for using Jupyter Notebook for the data exploration part so that we can get hands-on with Python and can gain deeper insights into the data. A wide range of visualizations, such as bar charts, scatter plots, maps, heatmaps, and more, are available in Tableau which will enable us to graphically depict our data, which will make it simpler to convey patterns and insights from data. So Tableau will be the best option for visualization in my opinion.

**Comment by Ashish8128** [comment]
Ok so lets use Jupyter Notebook and Tableau for data exploration and data visualization.

**Comment by varunbansal313** [comment]
Issue Closed. Will be using Jupyter Notebook and Tableau.
